client/k8s/k8s_test.go

```diff
@@ -1,263 +0,0 @@
-package k8s
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-	"k8s.io/client-go/rest"
-)
-
-func TestGetRawClientConfig(t *testing.T) {
```

Comment:
The operatorkit client has some tests but there are more here. I'll compare and see if any need adding.

We dropped some "features" like username and password from the operatorkit client ;)